src/driver/franka_plan_runner.cc

```diff
@@ -195,12 +195,12 @@ int FrankaPlanRunner::RunFranka() {
               "RunFranka: automaticErrorRecovery succeeded, out of Reflex mode,"
               " now in mode: {}.",
               utils::RobotModeToString(GetRobotMode()));
-        } catch (const franka::ControlException& ce) {
-          comm_interface_->SetDriverIsRunning(false, ce.what());
+        } catch (const franka::Exception& e) {
+          comm_interface_->SetDriverIsRunning(false, e.what());
           dexai::log()->warn(
-              "RunFranka: control exception in initialisation during automatic "
+              "RunFranka: caught exception in initialisation during automatic "
               "error recovery for Reflex mode: {}.",
-              ce.what());
+              e.what());
```

Comment:
can we add a separate catch for all non-CE?

Reply:
I don't think that a control exception is actually ever caught in this case... a catch-all seems fine, the recovery behavior would be the same for any type of exception (just retry to connect/recover)